day_7/index.js

```diff
@@ -1,320 +1,543 @@
-            arr[j] = '14';
-        } else if (arr[j] === 'f') {
-            arr[j] = '15';
-        }
-    }
-    firstNumber = arr.slice(0, 2);
-    secondNumber = arr.slice(2, 4);
-    thirdNumber = arr.slice(4);
-
-    let R = +firstNumber[0] * 16 ** 1 + +firstNumber[1];
-    let G = +secondNumber[0] * 16 ** 1 + +secondNumber[1];
-    let B = +thirdNumber[0] * 16 ** 1 + +thirdNumber[1];
-
-    return console.log('RGB (' + R + ',' + G + ',' + B + ')');
-
-
-}
-
-const convertRGBtoHex = (hexa) => {
-    let arr = [];
-    let arr2 = [];
-    let arr3 = [];
-    let str = String(hexa);
-    let str2 = ''
-    if (str[0] === '#') {
-        for (i = 1; i <= str.length - 1; i++) {
-            str2 += str[i];
-        }
-    }
-    arr = str.split(',')
-        /*  for (j = 0; j <= arr.length - 1; j++) {
-        if (arr[j] === 'a') {
-            arr[j] = '10';
-        } else if (arr[j] === 'b') {
-            arr[j] = '11';
-        } else if (arr[j] === 'c') {
-            arr[j] = '12';
-        } else if (arr[j] === 'd') {
-            arr[j] = '13';
-        } else if (arr[j] === 'e') {
-            arr[j] = '14';
-        } else if (arr[j] === 'f') {
-            arr[j] = '15';
-        }
-    }
-    firstNumber = arr.slice(0, 2);
-    secondNumber = arr.slice(2, 4);
-    thirdNumber = arr.slice(4);
- 
-*/
-    arr2[0] = Math.floor(+arr[0] / 16);
-    arr2[1] = +arr[0] % 16;
-    arr2[2] = Math.floor(+arr[1] / 16);
-    arr2[3] = +arr[1] % 16;
-    arr2[4] = Math.floor(+arr[2] / 16);
-    arr2[5] = +arr[2] % 16;
-
-    for (j = 0; j <= arr2.length - 1; j++) {
-        if (arr2[j] === 10) {
-            arr2[j] = 'a';
-        } else if (arr2[j] === 11) {
-            arr2[j] = 'b';
-        } else if (arr2[j] === 12) {
-            arr2[j] = 'c';
-        } else if (arr2[j] === 13) {
-            arr2[j] = 'd';
-        } else if (arr2[j] === 14) {
-            arr2[j] = 'e';
-        } else if (arr2[j] === 15) {
-            arr2[j] = 'f';
-        }
-    }
-    arr3[0] = String(arr2[0]) + String(arr2[1]) + String(arr2[2]) + String(arr2[3]) + String(arr2[4]) + String(arr2[5]);
-
-    return console.log(arr3);
-
-}
-
-
-const arrayOfHexaColors2 = (count) => {
-    let hexa = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'a', 'b', 'c', 'd', 'e', 'f'];
-    let array = [];
-
-
-    for (i = 0; i <= count; i++) {
-        let str = '';
-        for (j = 0; j <= 6; j++) {
-            str += hexa[Math.floor(Math.random() * 15 + 1)];
-        }
-        array.push('#' + str);
-
-    }
-
-    return console.log(array);
-}
-
-const generateColors = (numberName, count) => {
-    if (numberName == 'hex') {
-        arrayOfHexaColors2(count);
-    } else if (numberName == 'rgb') {
-        rgbColorGenerator(count);
-
-    }
-
-}
-
-generateColors('rgb', 2);
\ No newline at end of file
+const fullName = (firstName = 'Артур', lastName = 'Москвичёв') => {
+  let space = ' ';
+  let fullName = firstName + space + lastName;
+  return console.log(fullName);
+};
```

Comment:
А лучше вообще использовать template string и ебануть более наглядно (и однострочно)
```suggestion
const fullName = (firstName = 'Артур', lastName = 'Москвичёв') => 
    console.log(`${firstName} ${lastName}`);
```

Reply:
Вообще у тебя почему-то смесь двух отдельных задач:
```
1. Declare a function fullName and it prints out your full name.

2. Declare a function fullName and now it takes firstName, lastName as a parameter and it returns your full name.
```

В первой задаче ты сразу выводишь в функции на экран (`"and it prints out your full name"`, используя `console.log`) ,во второй же ты должен вернуть полное имя как строку (`"and it returns your full name"`). Разбей пожалуйста своё решение на корректные решения этих задач